### PR TITLE
Task/Add open in Taggit link

### PR DESCRIPTION
**Pull request by duckonomy**
## Overview: ##
Adds link to taggit for hazmapper.

## PR Status: ##

* [X] Ready.
* [ ] Work in Progress.
* [ ] Hold.

## Summary of Changes: ##
Adds link to hazmapper by utilizing taggit's `lastProj` stored in `localStorage` as subroutes seem to share `localStorage`.

## Testing Steps: ##
1. Unsure how to test locally at the moment... see **Notes**. However, observed behavior on production hazmapper/taggit.
 
## UI Photos:
Screen Shot 2023-04-19 at 3 18 33 AM

## Notes: ##
Related ticket: https://jira.tacc.utexas.edu/browse/WG-63
Currently as hazmapper and taggit are both hosted on same port, 4200,
there is no way to test shared behavior (e.g. linking between localStorage).

We might have to change port on taggit or hazmapper (this requires adding callbackUrl for that port).

Looking into this further, regarding `localStorage`, the values are not shared in subroutes if they have different ports (i.e. `localStorage` between http://localhost:4200 vs http://localhost:4201` is not shared).

We might have to just confirm the behavior on production and disregard this behavior locally unless we implement a workflow to serve hazmapper and taggit under same ports (via docker?)

There seem to be an issue on Taggit with the token not being invalidated properly from `localStorage` (and thus returning a 403)
Related to: https://github.com/taggit-tacc/taggit/pull/113#discussion_r1156557892
Possibly fixed with: https://github.com/taggit-tacc/taggit/pull/113/commits/b80c650db561889427f7805c476187dd35f36dc4

### CODE REVIEW

**Review by sophia-massie — APPROVED** [review]
LGTM with the added prefixes 👍

File: angular/src/app/components/users-panel/users-panel.component.html [hunk 1]
```diff
@@ -25,6 +25,11 @@ <h6 id="property-title">Description:</h6>
                     (nameChange)="changeProjectDescription($event)"
                 >
                 </app-edit-name-input>
+
+                <div style="display: flex; flex-direction: column; justify-content: center">
+                    <button class="button hollow" (click)="openTaggit()">Open Map in Taggit</button>
```
Comment (nathanfranklin) [inline angular/src/app/components/users-panel/users-panel.component.html:30]: what about: "Open map in Taggit" 


----------------


not for this PR but would good to have small Taggit logo in react version.

**Review by nathanfranklin — CHANGES_REQUESTED** [review]
Looks good to me.  👍  just two things to consider.

**Review by nathanfranklin — APPROVED** [review]
Excellent 👍

**Pull request merged**